src/components/ha-outlined-icon-button.ts

```diff
@@ -6,6 +6,11 @@ import { MdOutlinedIconButton } from "@material/web/iconbutton/outlined-icon-but
 @customElement("ha-outlined-icon-button")
 export class HaOutlinedIconButton extends MdOutlinedIconButton {
   static override styles = [
+    css`
+      .icon-button {
+        border-radius: var(--_container-shape);
+      }
+    `,
```

Comment:
Why isn't it with other styles (after super.styles) ?
Also, should we add a comment?

Reply:
Because `super.styles` should overrule it